### PR TITLE
Create sharath.json

### PR DESCRIPTION
**Pull request by thesct22**

### CODE REVIEW

File: domains/sharath.json [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (phenax) [inline domains/sharath.json:1]: Please add the required information to register the subdomain

**Review by phenax — CHANGES_REQUESTED** [review]

**Review by thesct22 — COMMENTED** [review]
Added necessary changes hopefully

**Review by thesct22 — COMMENTED** [review]
CNAME changed

File: domains/sharath.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Social media website project",
+  "repo": "https://github.com/thesct22",
+  "owner": {
+    "username": "thesct22",
+    "email": "sharathct22@gmail.com"
+  },
+  "record": {
+    "CNAME":"http://34.67.228.107.xip.io"
```
Comment (phenax) [inline domains/sharath.json:9]: This is not a valid cname.

Maybe this is what you wanted to do?

```suggestion
    "CNAME":"34.67.228.107.xip.io"
```

**Review by phenax — CHANGES_REQUESTED** [review]

File: domains/sharath.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Social media website project",
+  "repo": "https://github.com/thesct22",
+  "owner": {
+    "username": "thesct22",
+    "email": "sharathct22@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://thesct22.github.io/",
```
Comment (phenax) [inline domains/sharath.json:9]:
```suggestion
    "CNAME": "thesct22.github.io",
```

**Review by phenax — CHANGES_REQUESTED** [review]

**Review by phenax — APPROVED** [review]

**Pull request merged**